content/sigs/docs/index.adoc

```diff
@@ -54,6 +54,8 @@ participants:
 - name: "Zhao Xiaojie (Rick)"
   id: "surenpi"
   github: "LinuxSuRen"
+opengraph:
+  image: /images/logos/needs-you/Jenkins_Needs_You-02.png
```

Comment:
That's a nitpick, but I'd keep this together with the `logo:` in the YAML as they are virtually the same, rather than putting it after long list of participants.

Reply:
Thanks.  Added the change in a831c25b3bc3022e62bf63d18d00ecf9aedbbdec